.github/workflows/ci.yml

```diff
@@ -1,34 +1,26 @@
 name: CI
-
 on:
   push:
     branches: [master]
   pull_request:
     branches: [master]
-
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-
       - uses: purescript-contrib/setup-purescript@main
-        with:
-          purescript: "0.14.0-rc3"
-
+        with: {}
```

Comment:
This should omit the “with” altogether

Reply:
I did not manage to do this with `yq` but the following works 🎉

```
jq '.jobs.build.steps |= map(del(select(.uses | test("purescript-contrib/setup-purescript")?) | .with.purescript) | with_entries(select(.value | length > 0)))'
```

(at the cost of departing further away from our current formatting).